src/AppBundle/Entity/BackupSchedule.php

```diff
@@ -317,8 +325,9 @@ public function getShellCommands()
                 lxc snapshot ' . $container->getName() . '/"$r" \n
                 f=$(lxc publish ' . $container->getName() . '/"$r") \n
                 fingerprint=${f##*: } \n
-                # Hier wird duplicity befehl aufgerufen
-                lxc delete test2/"$r" \n
+                # Hier wird duplicity befehl aufgerufen \n
```

Comment:
In der foreach müsste es $this->containers anstatt $containers sein und das erste $commandTexts hat einen typo, fehlt ein m